cmd/job-executor-service-initcontainer/main.go

```diff
@@ -112,25 +118,20 @@ func main() {
 		log.Fatalf("unable to create keptn API: %s", err)
 	}
 
-	useLocalFileSystem := false
-
-	// configure keptn options
-	if env.Env == "local" {
-		log.Println("env=local: Running with local filesystem to fetch resources")
-		useLocalFileSystem = true
-	}
-
 	// re-create the event from job-executor-service
-	eventProps := keptn.EventProperties{
-		Project:     env.Project,
-		Stage:       env.Stage,
-		Service:     env.Service,
-		GitCommitID: env.GitCommitID,
+	eventProps := &keptnv2.EventData{
+		Project: env.Project,
+		Stage:   env.Stage,
+		Service: env.Service,
 	}
 
-	configService := keptn.NewConfigService(useLocalFileSystem, eventProps, keptnAPI.Resources())
+	resourceService := keptn_interface.NewV1ResourceHandler(*eventProps, keptnAPI.Resources())
```

Comment:
This is missing `gitCommitID` - V1 resource handler probably doesn't support it?

Reply:
The `gitCommitID` that was passed to the constructor was not used. You have to set it whenever calling a function e.g. `GetJobConfig(gitCommitID string)`. Still, I added the `gitCommitID` to the `mountFiles` method, so this should be fixed.